horace_core/utilities/@sqw/symmetrise_sqw.m

```diff
@@ -146,6 +149,10 @@
 idx = find(side_dot > 0);
 coords_new([1:3], idx) = Reflec*coords_new([1:3], idx); % MP: (TODO) could potentially be optimized further
 clear 'side_dot'; % MP: not needed anymore
+% Testing the ranges transformation
```

Comment:
new, commented out code

Reply:
this is the part of the work on the ticket with has been left outside the PR. The ticket itself is about problems with symmetrisation, and here I've realized that other work has to be done before going any further. Left here as starting point for further exploration.